src/ios/Classes/SwiftSfmcPlugin.swift

```diff
@@ -178,7 +178,7 @@ public class SwiftSfmcPlugin: NSObject, FlutterPlugin {
         let appEndpoint = URL(string: sfmcURL)!
         
         // To override the Keycahin accessibility attribute
-        SFMCSdk.setKeychainAccessibleAttribute(accessibleAttribute: kSecAttrAccessibleWhenUnlockedThisDeviceOnly)
+        //SFMCSdk.setKeychainAccessibleAttribute(accessibleAttribute: kSecAttrAccessibleWhenUnlockedThisDeviceOnly)
```

Comment:
Commenting out this line needs a reference from SFMC Native SDK instruction, can you please provide me with a reference from  marketing cloud SDK?